test/e2e/storage/drivers/csi.go

```diff
@@ -490,7 +490,7 @@ func waitForCSIDriverRegistrationOnAllNodes(driverName string, cs clientset.Inte
 }
 
 func waitForCSIDriverRegistrationOnNode(nodeName string, driverName string, cs clientset.Interface) error {
-	const csiNodeRegisterTimeout = 1 * time.Minute
+	const csiNodeRegisterTimeout = 5 * time.Minute
```

Comment:
do we know what is the average time for this registration roughly?

Reply:
Good question, from the failing test logs, it seems like the driver containers started fairly quickly. So maybe increasing the timeout won't help and maybe there's some other problem.